src/sentry/static/sentry/app/components/events/contexts/device/types.tsx

```diff
@@ -1,34 +1,41 @@
 export enum DeviceKnownDataType {
-  NAME = 'name',
-  FAMILY = 'family',
-  RENDERED_MODEL = 'renderedModel',
-  MODEL = 'model',
-  MODEL_ID = 'model_id',
-  CPU_DESCRIPTION = 'cpu_description',
   ARCH = 'arch',
+  ARCHS = 'archs',
```

Comment:
is this just the json field name or how its shown?

if possible, can we display differently in the UI (how its shown and not the json field name)?

Like `Supported architectures` or something similar?

`archs` is in the wrong plural form, my typo a few months ago :(

Reply:
this is just the json field name. The UI currently displays :

```
case DeviceKnownDataType.ARCHS:
      return {
        subject: t('Architectures'),
        value: data.archs,
      };
```

should I update it to `Supported architectures` ?

Reply:
cool, if its already in the correct form, no need to change, ty